### ISSUE TITLE
Missing ResourcesEmptyIcon to complement the ResourcesFullIcon

### ISSUE THREAD

**Issue by gildub** [patternfly/patternfly-design#930]
@patternfly/react-icons has:
* `ResourcesAlmostEmptyIcon`
* `ResourcesAlmostFullIcon`
* `ResourcesFullIcon`

Meanwhile there are no `ResourcesEmptyIcon` to complete them.

This would be a great enhancement for us (github.com/konveyor/virt-ui) to be able to create a PipelineSummary component such as:
![mockup](https://user-images.githubusercontent.com/1901741/95519428-57223900-09c5-11eb-87a5-c8aba86c9912.png)
instead we have to use `ResourcesAlmostEmptyIcon`:
![current](https://user-images.githubusercontent.com/1901741/95519438-5a1d2980-09c5-11eb-85f8-b9d29a52626a.png)

Currently using "@patternfly/react-icons": "^4.7.6"

**Comment by gildub** [comment]
@mcarrano, thanks for following-up.
The project is for the Migration Toolkit for Virtualization GUI (link in the description).
The time-frame is really tied as we have a code freeze for Beta in matter of few weeks.

**Comment by mcarrano** [comment]
@gildub this is something that I think we should ask, but it will be difficult to complete in time for your beta release. I discussed this with @mturley and @vconzola and they suggest that you stick with the `ResourcesAlmostEmptyIcon` for now. We will get this into the queue to add to our core icon set first and then it can be added to react-icons.

**Comment by bdellasc** [comment]
I don't think it was needed when the set was originally done, but it probably makes sense to add it

**Comment by mturley** [comment]
Since it would just be a plain grey circle, I might just throw together a quick and dirty CSS-based solution to render something that matches the size and stroke width of the other icons. Definitely not a priority for us, but nice to have.

**Comment by bdellasc** [comment]
If we were going to do that, I'd rather we just do it consistently with SVGs using the process Gina and I put together here (see Scenario one): https://docs.google.com/document/d/1b023B1QnCiYuoIvDJ1aUMZwk9Yk12drfmDuCfL44yyw/edit?usp=sharing

This would actually be a good test run for that procedure to make sure it's complete/accurate

**Comment by mturley** [comment]
Oh I just meant as a temporary placeholder in my product. but yeah, true, I should do it that way either way

**Comment by mcarrano** [comment]
I've opened a new issue here to add this icon to the core icon set. That will need to happen before adding to React-Icons. https://github.com/patternfly/patternfly/issues/3624

**Comment by bdellasc** [comment]
@evwilkin yes, we should add it to the set and add the appropriate description to the table. If it makes it easier, I can create that matching circle in SVG form, just so we're consistent with the other icons, then post the file here.

**Comment by bdellasc** [comment]
@evwilkin, here's the SVG file for Resources Empty (I used the same circle element from the other resource state icons):
[resources-empty.zip](https://github.com/patternfly/patternfly-design/files/5617086/resources-empty.zip)

**Comment by mcoker** [comment]
@bdellasc is that any different than the icon on https://patternfly-react.surge.sh/icons? 

<img width="150" alt="Screen Shot 2020-11-30 at 10 43 17 AM" src="https://user-images.githubusercontent.com/35148959/100637817-eb8b8680-32f8-11eb-83a6-a7b59d999da2.png">

**Comment by bdellasc** [comment]
@mcoker I'm not sure/can't tell by looking at it - looking back at the thread here, @mturley mentioned throwing together this icon as a stopgap. I was just following up with an actual SVG exported from our Sketch source file (like we've done with all the others). I used the same circle that is used in the other status icons, so it matches perfectly with all the other status icons. Just wanted to follow up, get the source set up on the visual design end so it's consistent with the other icons we've done, and add it to our other icon source materials.

**Comment by mturley** [comment]
Thanks @bdellasc . I didn't end up using a stopgap since we have the real `ResourcesEmptyIcon` available in the package now as of https://github.com/patternfly/patternfly/issues/3624. I agree it should be added to the site. @redallen do you know what we need to do there?

**Comment by redallen** [comment]
To add it to patternfly.org we need to add an entry to https://github.com/patternfly/patternfly-org/blob/master/packages/v4/src/content/design-guidelines/styles/icons/icons.js.

**Comment by mturley** [comment]
@redallen thanks. done. https://github.com/patternfly/patternfly-org/pull/2296
